requirements.txt

```diff
@@ -2,3 +2,4 @@ Sphinx
 sphinx_rtd_theme
 myst_nb
 sphinx-lesson
+sphinx_tabs
```

Comment:
```suggestion
```
same.

Reply:
OK I will rebase code and throw that commit away.